packages/api/src/domain/medical/document-query.ts

```diff
@@ -57,3 +51,19 @@ export function getStatusFromProgress(
   const isConversionCompleted = (successful ?? 0) + (errors ?? 0) >= (total ?? 0);
   return isConversionCompleted ? "completed" : "processing";
 }
+
+export function isProgressEqual(a?: Progress, b?: Progress): boolean {
```

Comment:
moved isProgressEqual and isDocumentQueryProgressEqual to here since they are utility functions for the dataypes defined in this file